CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.4.0] 2019-09-24
+- Add message to known errors link if pushing to catalog or collection fails.
+
+## [0.4.0] 2019-09-17
```

Comment:
We are missing the link for this at the bottom of this file, their is a duplicate [0.3.0].

Reply:
Solved 0.3.0 error link.
Not sure why we are missing a lot of information about the unreleased version in comparison with 0.4.0 but the link should be still the same isn´t it?

[Unreleased]: https://github.com/giantswarm/architect-orb/compare/v0.4.0...HEAD

Reply:
I don't either, but this look good now. Thanks for catching this.